core/spec.md

```diff
@@ -2728,6 +2741,18 @@ The following describes the attributes of Registry model:
   - A value of `true` indicates that Resource of this type supports a separate
     document to be associated with it.
 
+- `groups.resources.singleversionroot`
+    - Type: Boolean (`true` or `false`, case-sensitive).
+    - OPTIONAL.
+    - Indicates whether Resources of this type can have multiple Versions
+      that have their `ancestor` attribute set to itself.
```

Comment:
...can have multiple Versions
that represent roots of an ancestor tree, as indicated by
the Version's `ancestor` attribute value being the same as
its `versionid` attribute.
- - -
I think it's better to refer to this as "having multiple roots" instead of "things pointing to themselves", because 1) it'll align with the name of attribute and 2) I think the mental model of what "pointing to itself" means is an important think to convey... meaning, the key point is "multiple roots", and the self-pointer is just how it's expressed.  IOW we want people think "multi-roots is bad", not "more than one self-pointer is bad".